extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/NamespaceLookupExtractorFactory.java

```diff
@@ -125,7 +131,12 @@ public boolean start()
   public boolean close()
   {
     final Lock writeLock = startStopSync.writeLock();
-    writeLock.lock();
```

Comment:
whoops, wrong class, I retract my last statement and am looking at this comment.